pom.xml

```diff
@@ -70,12 +62,6 @@
       <version>${commonreportsVersion}</version>
       <type>jar</type>
     </dependency>
-    <dependency>
-      <groupId>org.openmrs.module</groupId>
-      <artifactId>referencedemodata-omod</artifactId>
-      <version>${referencedemodataVersion}</version>
-      <type>jar</type>
-    </dependency>
```

Comment:
Are we sure we want to stop overriding Reference Demo Data?

Reply:
There are some fixes already on the Reference Demo Data module we can safely depend on especially that this cab be good in dev and for close monitoring of bugs should they occur.